### PR TITLE
java-script

### PR DESCRIPTION
**Pull request by AnsarIbrahim**
### This PR adds the following sections :
- toogle
- mobile-version

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (ShadyShawkat) [inline .eslintrc.json:1]: - [ ] Please make sure that the JS linters are setup correctly, as per [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=For%20this%20project%2C%20you%20should%20add%20the%20JavaScript%20%2B%20HTML%20%2B%20CSS%20linter.) requirement.

File: index.html [hunk 2]
```diff
@@ -12,23 +12,39 @@
...
-                <span class="material-symbols-outlined">
-                    menu
-                </span>
+                <nav>
+                    <ul>
+                        <li><a href="#Portfolio">Portfolio</a></li>
+                        <li><a href="#about">About </a></li>
+                        <li><a href="#contact">Contact</a></li>
+                    </ul>
+                </nav>
```
Comment (ShadyShawkat) [inline index.html:31]: - [ ] Please make sure that the design on the desktop view is still consistent with the original design.
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bobb-Rob — APPROVED** [review]
## STATUS: APPROVED ✔️ 🍾 🍾 🍾 🥳 🥳 🥳 

Hi @AnsarIbrahim , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**